doc/GoogleSheetsConnector.md

```diff
@@ -134,8 +135,21 @@ myGoogleSheetsConnector.on({sheetIdOrTitle: 0}, myHandler)
 
 /** Check for changes only in a specific sheet by title and every 30 seconds */
 myGoogleSheetsConnector.on({sheetIdOrTitle: 'Sheet1', interval: 30 * 1000}, myHandler)
+
+/** Check for changes using a key column for more accuracy, see Known Issues below */
+myGoogleSheetsConnector.on({keyColumn: 'Identifier'}, myHandler)
 ```
 
+##### Known Issues:
+Since Google Sheets does not provide rows identifiers, it is hard to identify which events caused the rows changes,for example when deleting a row which is not at the bottom of the spreadsheet, the rows below are shifted up. This can be interpritated as an update of all the rows below including the "deleted" row and a deletion of the last row.
```

Comment:
`which events caused the changes, for example` - note missing space before comma